barrels/Semicircles/Semicircles.mc

```diff
@@ -0,0 +1,257 @@
+//
+// Copyright 2017 by Garmin Ltd. or its subsidiaries.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using Toybox.Position;
+using Toybox.Lang;
+using Toybox.Math;
+using Toybox.System;
+using Toybox.Test;
+
+module Semicircles {
```

Comment:
Do we need to have module level documentation for Monkey Barrels we're publishing?

Reply:
Probably not a bad idea to at least re-state the goal of the module